script/downtime.rb

```diff
@@ -4,9 +4,9 @@
 require 'uri'
 
 ENVIRONMENTS = {
-    'dev' => 'dev-adp.dsd.io',
-    'sandbox' => 'api-sandbox-adp.dsd.io',
-    'staging' => 'staging-adp.dsd.io',
+    'dev' => 'dev.claim-crown-court-defence.service.justice.gov.uk',
```

Comment:
As you are updating the domain, you may wish to update `lib/tasks/rake_helpers/load_test_runner.rb:40`